crates/router/src/consts.rs

```diff
@@ -27,3 +27,7 @@ pub(crate) const BASE64_ENGINE_URL_SAFE: base64::engine::GeneralPurpose =
 
 pub(crate) const API_KEY_LENGTH: usize = 64;
 pub(crate) const PUB_SUB_CHANNEL: &str = "hyperswitch_invalidate";
+
+// Apple Pay validation url
+pub(crate) const APPLEPAY_VALIDATION_URL: &str =
```

Comment:
Move the url to env

Reply:
I don't have the access to conf at connector level.